Add Markdown Files For Guides/Tutorials

Jdubrick opened the pull request:
As per our discussions today in the parking lot I converted the guides located below to Markdown files that are accessible to users external to Red Hat.

- https://docs.google.com/document/d/193uwS-uji9LTh4Npax5ABkHZnZzkLns9sN0c2ZxNZwc/edit?usp=sharing
- https://docs.google.com/document/d/15PWtMhdspdaf_UtzEvywA4KA5eUB7Y3wB8TC92WI4as/edit?usp=sharing

Jdubrick committed:
add md files for guides/tutorials

Signed-off-by: Jordan Dubrick <jdubrick@redhat.com>

thepetk commented on tutorials/model-guide.md:
```diff
@@ -0,0 +1,120 @@
+
+### Model Usage - code/usage-examples.md
+
+This document should aim to provide users with the information required to consume the model. For example if the model has a Python package that can be imported then it would be important to provide examples of different actions you can perform.
```
```suggestion
This document should aim to provide users with the information required to consume the model. For example, if the model has a Python package that can be imported then it would be important to provide examples of different actions you can perform.
```

thepetk commented on tutorials/model-guide.md:
```diff
@@ -0,0 +1,120 @@
+
+### Directory Format & Best Practices
+
+When looking to add Tech Docs to your resource there are a few practices you should adhere to, this will help ensure that everything is rendered correctly:
```
```suggestion
When looking to add Tech Docs to your resource there are a few practices you should adhere to; this will help ensure that everything is rendered correctly:
```

thepetk commented on tutorials/model-server-guide.md:
```diff
@@ -0,0 +1,90 @@
+
+- All Markdown files should be referenced within mkdocs.yaml, for file path you should reference the .md files as if mkdocs.yaml is the root:
+
+![Example Structure Img](../assets/mkdocs-structure-example.png)
```
I think the screenshot could be wider here:
![image](https://github.com/user-attachments/assets/c02262ba-9e4c-4212-904b-d76df955bd9c)

thepetk replied:
We could also use yaml code?

thepetk reviewed (commented):
Only some optional & minor comments.

In general it looks good to me and very useful guide.

Jdubrick committed:
address review comments

Signed-off-by: Jordan Dubrick <jdubrick@redhat.com>

Jdubrick commented:
Thanks for the review @thepetk, I addressed your comments as part of https://github.com/redhat-ai-dev/model-catalog-example/pull/5/commits/36dd91c443bad15ee3c1643ed380c5d966842080

thepetk commented:
> Thanks for the review @thepetk, I addressed your comments as part of [36dd91c](https://github.com/redhat-ai-dev/model-catalog-example/commit/36dd91c443bad15ee3c1643ed380c5d966842080)

I think we can remove the screenshot now right? Or is used elsewhere?

Jdubrick committed:
remove unused screenshot asset

Signed-off-by: Jordan Dubrick <jdubrick@redhat.com>

Jdubrick commented:
> > Thanks for the review @thepetk, I addressed your comments as part of [36dd91c](https://github.com/redhat-ai-dev/model-catalog-example/commit/36dd91c443bad15ee3c1643ed380c5d966842080)
> 
> I think we can remove the screenshot now right? Or is used elsewhere?

Yup I forgot oops
